src/dotnet/UI.Blazor/Components/AuthorBadge/AuthorName.razor

```diff
@@ -0,0 +1,35 @@
+@namespace ActualChat.UI.Blazor.Components
+@using ActualChat.Users
+
+@if (ReferenceEquals(Author, Users.Author.None)) {
```

Comment:
I noticed that in AuthorCircle you also compara Author against null.
Is there a reason why you don't do it here?

Reply:
I will do the same